internal/serve/middleware/middleware_test.go

```diff
@@ -19,11 +20,12 @@ import (
 
 func TestSignatureMiddleware(t *testing.T) {
 	signatureVerifierMock := auth.MockSignatureVerifier{}
+	appTrackerMock := apptracker.MockAppTracker{}
```

Comment:
Please can you move this line below the `defer signatureVerifierMock.AssertExpectations(t)`.

```suggestion
	appTrackerMock := apptracker.MockAppTracker{}
	defer appTrackerMock.AssertExpectations(t)
```

Reply:
Done